src/main/resources/templates/experiments.html

```diff
@@ -104,7 +107,28 @@ <h4 class="modal-title" id="expId"><span th:text="#{experiment.table.header.deta
                     </div>
                 </div> <!-- panel primary -->
             </div> <!-- row -->
+            <tr>
+                <td height="1000"></td>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this deprecated "height" attribute. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)